cmd/release-controller/sync_tags.go

```diff
@@ -96,10 +96,23 @@ func (c *Controller) replaceReleaseTagWithNext(release *Release, tag *imagev1.Ta
 
 func (c *Controller) removeReleaseTags(release *Release, removeTags []*imagev1.TagReference) error {
 	for _, tag := range removeTags {
-		// use delete imagestreamtag so that status tags are removed as well
-		klog.V(2).Infof("Removing release tag %s", tag.Name)
-		if err := c.imageClient.ImageStreamTags(release.Target.Namespace).Delete(context.TODO(), fmt.Sprintf("%s:%s", release.Target.Name, tag.Name), metav1.DeleteOptions{}); err != nil {
-			if !errors.IsNotFound(err) {
+		// indicate by annotation that the isTag should be deleted
+		klog.V(2).Infof("Soft-removing (annotating) release tag %s", tag.Name)
+		ctx := context.TODO()
+		name := fmt.Sprintf("%s:%s", release.Target.Name, tag.Name)
```

Comment:
There is (should be) a function for this, don't implement yourself.

Reply:
I cannot grep (with `grep -irn '%s:%s' .`) out a reusable function for this.

Reply:
Nm, it didn't get added :(.  Disregard this comment for now.